src/linker/configuration.py

```diff
@@ -19,41 +11,18 @@ class Config:
     """
 
     def __init__(
-        self, pipeline_specification: str, computing_environment: Path, input_data: str
+        self, pipeline_specification: str, computing_environment: Union[None, str], input_data: str
```

Comment:
```suggestion
        self, pipeline_specification: str, computing_environment: Optional[str], input_data: str
```

Or is there a preference to not use `Optional`? I see this change below as well.

Reply:
I just don't think it matters much. Union[None, str] and Optional[str] are the same thing (Optional is mroe "modern"). In this case I actually prefer Union b/c it feels like click actually passes in None (which isn't probably precisely true)